src/receiver.rs

```diff
@@ -195,6 +200,11 @@ fn receive_packets<T>(recv: PacketReceiver, kill: Arc<AtomicBool>, send: Sender<
         trace!("Waiting on packet at {}...", recv);
         let (msg_bytes, addr) = match recv.recv_pckt() {
             Ok((bytes, addr)) => (bytes, addr),
+            Err(ref err) if err.kind() == io::ErrorKind::WouldBlock => {
```

Comment:
OS X and Linux return `WouldBlock` (see: https://is.gd/TFOY1G). There should probably be a rust issue for this.

I suppose for the moment the check should be for both.